iceoryx_hoofs/posix/utility/source/posix_scheduler.cpp

```diff
@@ -47,16 +39,8 @@ int32_t getSchedulerPriorityMaximum(const Scheduler scheduler) noexcept
     auto result = IOX_POSIX_CALL(sched_get_priority_max)(static_cast<int>(scheduler)).failureReturnValue(-1).evaluate();
     if (result.has_error())
     {
-        IOX_LOG(ERROR,
-                "The \"sched_get_priority_max\" should never fail. This can only be caused by an internal logic "
-                "error or a non posix compliant system.");
-
-        // NOLINTJUSTIFICATION Required to provide an error message to the user
-        // NOLINTNEXTLINE(hicpp-no-array-decay,cppcoreguidelines-pro-bounds-array-to-pointer-decay)
-        IOX_ENSURES(false
-                    && "This should never happen! Either the system is not posix compliant or and invalid integer was "
-                       "casted to the enum class Scheduler.");
-        return -1;
+        IOX_PANIC("The \"sched_get_priority_min\" should never fail! Either the system is not POSIX compliant or and "
```

Comment:
```suggestion
        IOX_PANIC("The \"sched_get_priority_max\" should never fail! Either the system is not POSIX compliant or an "
```

Reply:
This used call is actually `sched_get_priority_max`.

Reply:
Just wanted to see if you pay attention ;)